src/trix/models/attachment.coffee

```diff
@@ -117,7 +118,11 @@ class Trix.Attachment extends Trix.Object
       @uploadProgressDelegate?.attachmentDidChangeUploadProgress?(this)
 
   toJSON: ->
-    @getAttributes()
+    attributes = @getAttributes()
+    if @getContentType() == 'text/html' && @getURL()
+      delete attributes.content
```

Comment:
This is preventing it from storing `content` that I fetched from the URL and forces it to always refetch the latest content.